doc/simulation/inspector/src/entry.py

```
@@ -6,6 +6,7 @@
 from cpu_usage import *
 from memory_usage import *
 from pvc_usage import *
+from datetime import datetime, timedelta  # re-import after wildcard imports to avoid shadowing
```

Ruff (0.15.2)

[error] Redefinition of unused `datetime` from line 1: `datetime` redefined here

Remove definition: `datetime`

(F811)